.github/workflows/downstream_repo_trigger.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Downstream Repo Checks'
+on:
+  push:
+    branches:
+      - nightly
```

Comment:
Todo: we could run it even on every push but before doing it I need to improve CI cache in `sov-rollup-starter`